mgechev/angular-seed
# Issue with scss build prod

I found an issue with `.css.map` files after `build.prod / build.prod.rollup.aot`.

<img width="1679" alt="bug" src="https://user-images.githubusercontent.com/7300673/33036038-f0200020-ce35-11e7-8232-723ec90c7f1c.png">


Test case:
1. Convert all files to `.scss`.
2. Set flag `ENABLE_SCSS = true` in `project.config.ts`.
3. Run `build.prod.rollup.aot` and verify or `serve.prod.rollup.aot`.

Issue resolved.
